src/tests/stdio/Mybuild

```diff
@@ -22,3 +22,14 @@ module fgets_test {
 	depends embox.compat.libc.all
 	depends embox.framework.LibFramework
 }
+
+module fileIOErrorTest {
+	source "fileIOErrorTest.c"
+	
+	@InitFS
+	source  "fileIOErrorTest.txt"
```

Comment:
Our code styles assume underscores in function and files namings -https://github.com/embox/embox/wiki/Code-Style

Can you please fix this?